src/questions/aave-other-features.yaml

```diff
@@ -251,5 +251,153 @@
+      key: C
+    - content: Facilitator is an alias for liquidator
+      key: D
+- uuid: b850e217-7dd0-4bdb-8766-9cec9d667554
+  type: SingleChoice
+  content: What is GHO?
+  hint: noHint
+  explanation: GHO is a collateral-backed stablecoin native to Aave, it is decentralized and pegged to USD
+  answerKeys:
+    - C
+  subTopics:
+    - aave-gho
+  difficultyLevel: "Low"
+  choices:
+    - content: A collateral-backed stablecoin native to Aave, it is centralized and collateral backed pegged to USD
+      key: A
+    - content: A fiat-backed stablecoin native to Aave, it is decentralized and pegged to USD
+      key: B
+    - content: A collateral-backed stablecoin native to Aave, it is decentralized and pegged to USD
+      key: C
+    - content: A collateral-backed stablecoin, not native to Aave, it is centralized and pegged to USD
+      key: D
+
+- uuid: 865bf61c-1cf6-494f-a537-74c8140031db
+  type: MultipleChoice
+  content: Which of these are true regarding GHO’s scalability?
+  hint: noHint
+  explanation: Using portals, GHO can be minted on Ethereum and be transferred to other chains easily. The non-crypto native user base prefers stablecoins.
+   Penetration into L2 chains would be desirable as they are gaining popularity amongst a vast majority of general users. 
+  answerKeys:
+    - A
+    - B
+    - C
+  subTopics:
+    - aave-gho
+  difficultyLevel: "Medium"
+  choices:
+    - content: Using portals, GHO can be minted on Ethereum and be transferred to other chains easily
+      key: A
+    - content: The non-crypto native user base prefers stablecoins
+      key: B
+    - content: The non-crypto native user base does not prefer stablecoins
+      key: C
+    - content: Penetration into L2 chains would be desirable as they are gaining popularity amongst a vast majority of general users
+      key: D
+- uuid: 5598e841-0111-4515-8600-9525469fe1e5
+  type: MultipleChoice
+  content: Which of these contribute to the safety of GHO’s use?
+  hint: noHint
+  explanation: It is minted on Ethereum and supplied to other chains, since Ethereum is much safer. A supply borrow cap can be levied to further improve its security.
+    Only authorized facilitators can mint and burn GHO trustlessly. It is a stablecoin, and it has a stable interest rate which makes it relatively more predictable than usual crypto assets, these factors 
+    improve the security of GHO.
+  answerKeys:
+    - A
+    - B
+    - C
+    - D
+  subTopics:
+    - aave-gho
+  difficultyLevel: "Medium"
+  choices:
+    - content: A supply borrow cap can be levied to further improve its security
+      key: A
+    - content: It is minted on Ethereum and supplied to other chains, since Ethereum is much safer
+      key: B
+    - content: Only authorized facilitators can mint and burn GHO trustlessly
+      key: C
+    - content: It is a stablecoin, and it has a stable interest rate which makes it relatively more predictable than usual crypto assets
```

Comment:
`It is a stablecoin, and it has a stable interest rate which makes it relatively more predictable than usual crypto assets` change this to an incorrect one as this is not related to security, but the statement is true